src/main/java/frc/robot/subsystems/Manager.java

```diff
@@ -176,8 +186,17 @@ public void periodic() {
         
         if (robot.secondaryController.getXButtonPressed()) {
             state = ManagerStates.IDLE;
+            robot.drive.fieldRel();
             reset();
         }
+
+        if (robot.secondaryController.getStartButtonPressed()) {
```

Comment:
Shouldn't this state transition be in Drive? It seems to have more to do with Drive, at least. I think it makes more sense to store targetPose in Drive, and drive can call Manager reset to put it into idle. also, as discussed in earlier comment, you should def get rid of `completedScore` and just transition immediately after you reach the correct position.